flatbuffers/MarketDataSnapshot.fbs

```diff
@@ -56,4 +56,8 @@ table MarketDataSnapshot {
     bid: [MDEntry];
     /// Offer market data entries.
     offer: [MDEntry];
+    // Decimal places for price
+    price_dp: int16;
```

Comment:
can we not get this from static?

Reply:
we have moved to using number of dp in the quote for most LPs. Some instruments can be quoted to different d.p depending on LP. https://reactivemarkets.atlassian.net/wiki/spaces/ENG/pages/973471745/LP+Full+Amount+Stack+Rounding

Reply:
yes but static is already by LP anyway.

Reply:
we could, but then you would need to maintain this information and keep it in sync with LPs and I think we want to avoid that

Reply:
We already have and use price_dp from static. It’s maintained.

Reply:
Hi Mark. Not, easily. I will call to explain when I have some time.